recipes/snakeviz/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "snakeviz" %}
+{% set version = "v2.0.0" %}
```

Comment:
Let's remove the `v` here.

Reply:
Happy to, though that is the tagged release "number" -- so there'll be a `v{{ version }}` in the source url

Reply:
Right yeah the URL is fine. We just want the `v` not in the official version.